### DISCUSSION TITLE
This user account is blocked. Contact registration@vercel.com for more information.

### DISCUSSION THREAD

**Discussion by aryakghosal** [vercel/community#1505] [Help] [answered]
### Summary

This is my first time signing in to Vercel and I can't Sign-In with my GitHub Account. 
It is giving me a message "This user account is blocked. Contact registration@vercel.com for more information."
Please Help!

**Comment by swarnava** [comment, marked as answer]
Hi, You need to reach out to registration@vercel.com to unlock your account as instructed.

**Reply by aryakghosal** [reply]
Thank You, I did the same, but haven't heard back from them... yet.

**Reply by swarnava** [reply]
We don't provide turnaround time for Hobby plan: https://vercel.com/pricing. However, I just replied to the email. Thank you!

**Reply by aryakghosal** [reply]
Thank You for your time :)
